semantic.d

```diff
@@ -11010,12 +11023,16 @@ mixin template Semantic(T) if(is(T==ReferenceAggregateDecl)){
 			}
 		}
 	Lfresh:
-		if(!set && decl.stc & STCoverride){
+		if(set) goto Lfresh3;
```

Comment:
While most of the errors "goto skips declaration" are false positives, it actually uncovers a bug here: "set" might have been checked without actually been initialized.

BTW: I avoided indenting the code due to the additional braces to minimize the actual changes. I suspect you also often use goto to avoid the indentation.